### DISCUSSION TITLE
Name and Logo

### DISCUSSION THREAD

**Discussion by ArtsphereOfficial** [Xed-Editor/Xed-Editor#43] [Ideas]
Change name of editor and replace current logo with a brand new logo(design) accordingly to new name of app.

**Comment by RohitKushvaha01** [comment]
> Change name of editor and replace current logo with a brand new logo(design) accordingly to new name of app.

That's a problem because I need to learn logo design first

**Reply by ArtsphereOfficial** [reply]
You can do it in Figma
